src/pages/RouteDefinitions.tsx

```diff
@@ -178,19 +178,13 @@ export const routes: RouteDefinition[] = [
   }),
   createRouteDefinition({
     path: '/add',
-    nestedPaths: [':currencyIdA', ':currencyIdA/:currencyIdB', ':currencyIdA/:currencyIdB/:feeAmount'],
-    getElement: () => <RedirectDuplicateTokenIds />,
-  }),
-
-  createRouteDefinition({
-    path: '/increase',
     nestedPaths: [
       ':currencyIdA',
       ':currencyIdA/:currencyIdB',
       ':currencyIdA/:currencyIdB/:feeAmount',
       ':currencyIdA/:currencyIdB/:feeAmount/:tokenId',
```

Comment:
`':currencyIdA/:currencyIdB/:feeAmount/:tokenId',` looks like it might be a diff behavior than before?

Reply:
it wasn't listed as a valid Path under `/add` before, but they both render the same component under the hood so it's supported.